lib/routes.js

```diff
@@ -88,22 +89,22 @@ exports.routes.push({
 
 exports.routes.push({
     method: 'GET',
-    path: '/tutorials/{tutorial?}',
+    path: '/tutorials/{lang}/{tutorial?}',
```

Comment:
great point @luanmuniz we definitely want to make sure existing links don't break. i'd be fine with using a query parameter to specify language also